app/models/user.rb

```diff
@@ -25,6 +27,24 @@ def self.digest string
     BCrypt::Password.create(string, cost: cost)
   end
 
+  def self.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute(:remember_digest, User.digest(remember_token))
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.nil?
+    BCrypt::Password.new(remember_digest).is_password?(remember_token)
+  end
+
+  def forget
+    update_attribute(:remember_digest, nil)
```

Comment:
rails có những cách nào để update data nhỉ ?

Reply:
Từ bản 2.5 trở đi thì có 2 cách là update và update_column:
+ update_attribute và update_attributes là 2 alias của update, update toàn bộ thông tin.
+ update_column là chỉ update 1 cột ạ.

Reply:
![image](https://user-images.githubusercontent.com/46867650/55168572-a1f2ac00-51a5-11e9-89d2-2b018b81b7d3.png)
Theo như hình thì có 4 loại là update_attribute, update_column, update và update_all ạ
Nhưng hôm nay em được chị Dương review và recommend là luôn dùng update cho mọi cái.